examples/apps/colorapp/src/colorteller/deploy.sh

```diff
@@ -27,9 +27,17 @@ ecr_login() {
     fi
 }
 
+describe_create_ecr_registry() {
```

Comment:
This will create the registry if it doesn't exist already. Otherwise we would try to upload the file and just fail instead.